trackomatic.js

```diff
@@ -15,6 +15,31 @@
  * limitations under the License.
 */
 
+var BODY = document.getElementsByTagName('body')[0];
```

Comment:
Oh, true — at first, I thought this was being pasted into GTM (in which case, it's fine) but if the file is being loaded directly, we gotta wrap in a SEAF.